package/debian/Dockerfile

```diff
@@ -42,15 +42,14 @@ RUN    apt-get update            \
         pkg-config               \
         python3                  \
         python3-all              \
+        python3-distutils        \
```

Comment:
This probably means that we need to put this in `package/debian/control?` as a `Build-Depend`?